projects/plugins/jetpack/extensions/blocks/tiled-gallery/utils/index.js

```diff
@@ -47,8 +51,7 @@ export function photonizedImgProps( img, galleryAtts = {} ) {
 	const { height, width } = img;
 	const { layoutStyle } = galleryAtts;
 
-	const photonImplementation =
-		isWpcomFilesUrl( url ) || true === isVIP() ? photonWpcomImage : photon;
+	const photonImplementation = true === isVIP() || isSimpleSite() ? photonWpcomImage : photon;
```

Comment:
We'll want the `photonWpcomImage` implementation, since the files are hosted on the WordPress.com infrastructure there. We do not need to have WordPress.com files processed by Photon on top of that.